Java/Classes.java

```diff
@@ -0,0 +1,123 @@
+
+/**
+ * OOPs (Object-Oriented Programming System)
+ * 
+ * Object means a real-world entity such as a pen, chair, table, computer,
```

Comment:
- Class definition is missing .
- No need of object definition here 
- Particular file should contains only particular definition.